cumulusci/cumulusci.yml

```diff
@@ -900,6 +927,7 @@ flows:
                 task: install_managed_beta
 
     release_beta:
+        group: Release Management
```

Comment:
How about Release Operations

Reply:
I was having it match the tasks group for this one. But I like Operations better. Should we change the tasks group name in a separate PR to match?

Reply:
@Br4nd0R Sure. It can go in this PR if you want.